tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/statistics/BinaryStatistics.java

```diff
@@ -36,6 +36,9 @@
 
   private Binary firstValue = new Binary("");
   private Binary lastValue = new Binary("");
+  private static final String BINARY_STATISTICS_NOT_SUPPORT_MIN = "Binary statistics does not support: min";
+  private static final String BINARY_STATISTICS_NOT_SUPPORT_MAX = "Binary statistics does not support: max";
+  private static final String BINARY_STATISTICS_NOT_SUPPORT_SUM = "Binary statistics does not support: sum";
```

Comment:
Use BINARY_STATS_UNSUPPORTED_MSG to represent "Binary statistics does not support: %s", 
and use String.format(BINARY_STATS_UNSUPPORTED_MSG, "min");